tests/drivers/eeprom/api/testcase.yaml

```diff
@@ -26,8 +30,6 @@ tests:
       - native_posix_64
       - native_sim
       - native_sim_64
-    integration_platforms:
-      - native_sim
```

Comment:
Same comment here as above, wht was this removed

Reply:
I've reverted